packages/react/src/components/Forms/DatePicker/DateRangePicker.tsx

```diff
@@ -67,6 +67,7 @@ const DateRangePicker = ({
   return (
```

Comment:
A11Y: https://www.loom.com/share/a59aebb2208844bdb09603b08a5ee22a

Reply:
Might bad, I haven't understood correctly your issue and partially fixed it. I gonna improve my fix.

Reply:
It should be fine, I let you test and resolve the comment.